compose/cli/docopt_command.py

```diff
@@ -34,6 +34,10 @@ def parse(self, argv):
         if docstring is None:
             raise NoSuchCommand(command, self)
 
+        # checks to see if the command is help and if so outputs top level command options
+        if command == 'help' and options['ARGS'] == []:
+            raise SystemExit(docstring + '\n-------------------\n\n' + command_help)
```

Comment:
@twitherspoon my comment is not python specific, it is about unix best practices. The `docker` command behaves like this. You can verify by checking the exit codes like this:

```bash
$ docker help ; echo $?  # exit code is 0 (OK)
$ docker nonsense ; echo $?  # exit code is 1 (error)
```

and the output channels with
```bash
$ docker help | grep container  # help lines containing "container" (stdout)
$ docker nonsense | grep container  # error message on stderr, lines without "container" appear
```